src/Microsoft.ML.Data/Scorers/PredictionTransformer.cs

```diff
@@ -595,33 +671,125 @@ private static VersionInfo GetVersionInfo()
     internal static class BinaryPredictionTransformer
     {
         public const string LoaderSignature = "BinaryPredXfer";
+        private const string DirModel = "Model"; // This should match "DirModel" in PredictionTransformerBase<>
```

Comment:
Can the existing `const string` be reused instead?

Reply:
I believe it can't without creating an instance of `PredictionTransformerBase<TModel>`. I tried other approaches but they didn't work. Do you know of a way to reuse the existing one in `PredictionTransformeBase<>` without making an instance?

Reply:
I also thought of creating a new class just to hold the DirModel string, and reuse it in all of the other classes (including the existing one in PredictionTransformerBase). But I don't know if creating a whole other class only for that would be ok.

Reply:
You don't need to create an instance to reference a `const` variable. Rather, the issue here is that since `PredictionTransformerBase<TModel>` is generic, you need to fill out the generic type in order to reference the `const`.

Making a new `static class` that is not generic is probably the best approach for this.

Reply:
In my latest update I've created an internal static class `PredictionTransformerBase ` only to hold this string.